src/components/LocationPage/Notifications.tsx

```diff
@@ -98,11 +105,29 @@ const NotificationArea: React.FC<{ projections: Projections }> = ({
         {notification === Notification.NYCCounty && (
           <NYCAggregationChangeCopy locationName={region.name} />
         )}
+
+        {notification === Notification.TXFeb2021Winter && (
+          <TXFeb2021WinterCopy locationName={region.name} />
+        )}
       </SectionColumn>
     </React.Fragment>
   );
 };
 
+const TXFeb2021WinterCopy: React.FC<{ locationName: string }> = ({
+  locationName,
+}) => {
+  return (
+    <Copy>
+      In February 2021, {locationName} experienced extreme winter weather that
```

Comment:
How sure are we that we want to say this for all counties in Texas?  It's a big state. Was all of Texas incapacitated, or just some regions?

At minimum perhaps we should say `Texas` instead of `{locationName}` to imply less precision.

Reply:
There will be variations in impact, but I wouldn't trust any Texas data for the rest of the month. I'm fine with changing it to say `Texas`.

Reply:
Changed.

Reply:
We should be worried about giving people over confidence that cases are continuing to decrease. Given delays in reports, decreased test seeking, and closed testing centers, it'll be hard to get a positivity PCR test regardless of the true infection rate.